src/modExcelToXML.bas

```diff
@@ -0,0 +1,169 @@
+Attribute VB_Name = "modExcelToXML"
+Option Explicit
+
+Const ms As Double = 0.000000011574
+
+Sub excelToXML()
+    
+    If Not changeFileExtension(ThisWorkbook.Path, ThisWorkbook.Name) Then
+            
+    End If
+        
+    If Not windowsUnZip(Left(ThisWorkbook.FullName, Len(ThisWorkbook.FullName) - 4) & "zip", ThisWorkbook.Path) Then
+    
+    End If
+        
+End Sub
+
+Function changeFileExtension(strPath As String, strFileName As String) As Boolean
+    
+    Dim FSO As New FileSystemObject
```

Comment:
Like most things in life mate "it depends" sorry to be so vague about it, I used to love using global variables and still do but there are some instances when it can trip you up

I see no reason why there can't be a `m_fso` as a module level fso